examples/hooks/components/layout/Tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { cx } from '../../cx';
+
+import './Tabs.css';
+
+export type TabProps = {
+  title: string;
+};
+
+export const Tabs = ({ children }) => {
+  const [currentTab, setCurrentTab] = useState(0);
+
+  return (
+    <div className="ais-Tabs">
+      <ul className="ais-Tabs-header">
+        {React.Children.map<React.ReactChild, React.ReactElement<TabProps>>(
+          children,
+          (child, index) => (
+            <li
+              className={cx(
+                'ais-Tabs-title',
+                currentTab === index && 'ais-Tabs-title--active'
```

Comment:
not sure if it's worth it for this example, but this probably should be following the ARIA specs for tabs. I guess something like `aria-active` should be used if we want to show the good example.

Reply:
I think it's worth it—some of our users might rely on it to use our demos, and it sets a good example.

[This markup](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html#sc1_label) should be fine.